multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1475,7 +1475,6 @@ def _fit(
                 log_every_n_steps=OmegaConf.select(config, "optimization.log_every_n_steps", default=10),
                 enable_progress_bar=enable_progress_bar,
                 fast_dev_run=config.env.fast_dev_run,
-                track_grad_norm=OmegaConf.select(config, "optimization.track_grad_norm", default=-1),
```

Comment:
Please update the doc correspondingly: https://github.com/autogluon/autogluon/blob/master/docs/tutorials/multimodal/advanced_topics/customization.ipynb?short_path=c0bd10d#L368

Reply:
BTW, as lightning has removed `track_grad_norm ` from trainer, do you know if lightning has some equivalent design for it?

Reply:
ref https://github.com/Lightning-AI/lightning/pull/16745 they suggest an explicit `log_dict`. any thoughts on this?

Reply:
> Please update the doc correspondingly: https://github.com/autogluon/autogluon/blob/master/docs/tutorials/multimodal/advanced_topics/customization.ipynb?short_path=c0bd10d#L368 We can remove `track_grad_norm ` from the doc.

https://github.com/autogluon/autogluon/pull/3190/commits/88d06643d520f7daec300b002429cad4485a860b